### PR TITLE
More Preprocessing

### PR DESCRIPTION
**Pull request by greatsharma**
Added `MaxAbsScaler`, `RobustScaler`, `AbsMax` and `RollingAbsMax`.

Also added support of `with_mean` and `with_std` to `StandardScaler` as there may be the case in which user only want to do one of the two things and also it makes the class more close and consistent with `sklearn.StandardScaler`.

### CODE REVIEW

File: creme/preprocessing/scale.py [hunk 1]
```diff
@@ -25,6 +28,10 @@ class StandardScaler(base.Transformer):
     different than when using scikit-learn but this doesn't seem to have any impact on learning
     performance.
 
+    Parameters:
+        with_mean (boolean): Whether to centre the data before scaling. Defaults to ``True``.
```
Comment (MaxHalford) [inline creme/preprocessing/scale.py]: Should be `bool` not `boolean`.

**Comment by MaxHalford** [comment]
Good stuff @greatsharma :)

File: creme/preprocessing/scale.py (hunk 1 above)
Comment (greatsharma) [reply creme/preprocessing/scale.py]: I corrected it now

**Comment by MaxHalford** [comment]
You now need to check out Travis and see why some of the tests failed.

**Comment by greatsharma** [comment]
> You now need to check out Travis and see why some of the tests failed.

@MaxHalford I fixed them now. I think it would be nice if a code formatter is added to this library like `BLACK`. What's your thought on this?

**Comment by MaxHalford** [comment]
Good work. Yeah we definitely have to set some rules in stone regarding code style. I don't really like `black` for many reasons. I think that a let invasive solution such as a `.pylintrc` file would be great :)

**Pull request merged**